Src/App.js

```diff
@@ -3,13 +3,41 @@
        import ReactDOM from 'react-dom/client';
        
        /* Hello World Using JS!!*/
```

Comment:
the folder name generally starts with a small letter and component name starts with capital letter, .css file also starts with small letter